package.json

```diff
@@ -3,7 +3,7 @@
   "description": "OONI Probe Desktop app",
   "author": "Open Observatory of Network Interference (OONI) <contact@openobservatory.org>",
   "productName": "OONI Probe",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
We should not bump the version number until we are ready to make a release.

Perhaps a good practice is to, after we make a release, replace the version number with `${probeVersion}-dev`